tests/testthat/test-with-timer.R

```diff
@@ -14,3 +14,20 @@ test_that("with_timer", {
     20
   }), 20), "^00:00:00\\.\\d{3,3}\\s$")
 })
+
+test_that("with_timer srcref = TRUE", {
+  fun <- function() {
+    Sys.sleep(0.1)
+    10
+  }
+  expect_message(expect_identical(with_timer(fun(), srcref = TRUE), 10), "^00:00:00\\.\\d{3,3} \\[.*:.*\\]\\s$")
```

Comment:
```suggestion
  # Can't do .+ to match file:line because in some settings no file/line info is present
  expect_message(expect_identical(with_timer(fun(), srcref = TRUE), 10), "^00:00:00\\.\\d{3,3} \\[.*:.*\\]\\s$")
```